client/app/review/create/(components)/(Form)/Form.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+
+import { useState } from 'react';
+
+import { NewReview } from 'types/new_review';
+import TextArea from '@components/UI/TextArea';
+
+import styles from './Form.module.scss';
+import Dropdown from '@components/UI/Dropdown';
+import { Option } from 'react-dropdown';
+
+async function sendFormData(newReview: NewReview) {
+    await fetch('http://localhost:8000/api/review/create', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(newReview)
+    })
+        .then((res) => res.json())
+        .then((data) => {
+            console.log(data);
+        });
+}
+
+export default function Form() {
+    const [reviewBody, setReviewBody] = useState('');
+
+    const gradeOptions: Option[] = [
```

Comment:
I'm not sure but will other parts of the code need this variable? if so we should make it global so everywhere can access it, or make it an export

Reply:
Again apologies, this is unfinalised code, it should not be part of this branch